html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="index.html" method="post">
```

Comment:
You should not have an individual `form` element around each `input`, there should be one `form` element to wrap all the content of that form.